cypress/support/pages/contactForm.pageObject.js

```diff
@@ -31,8 +35,24 @@ class ContactFormPageObject extends PageObject {
     this.messageField.type(message);
   }
 
-  clickOnSendMessageBtn() {
-    this.sendMessageBtn.click();
+  typeCountry(country) {
+    this.countryField.type(country, { force: true });
+  }
+
+  typeCity(city) {
+    this.cityField.type(city, { force: true });
+  }
+
+  typeCard(card) {
+    this.cardField.type(card, { force: true });
+  }
+
+  typeMonth(month) {
+    this.monthField.type(month, { force: true });
```

Comment:
The function 'typeMonth' is missing the super call to the 'type' method. All inputs should be passed to a super method to handle typing into the field.